src/config/dynamic-theme-fixes.config

```diff
@@ -3566,6 +3566,16 @@ INVERT
 
 ================================
 
+firstcontributions.github.io
+
+CSS
+.App,
+.topnav {
+    background-color: var(--darkreader-neutral-background);
```

Comment:
Hmm, might be better of changing to a darker blue.

Reply:
3a861b7621d80f3869b8c46eaddc21e8cfa3a4de